src/amber/cli/templates/error/spec/controllers/error_controller_spec.cr.ecr

```diff
@@ -0,0 +1,57 @@
+require "./spec_helper"
+
+# Controller Unit Test
```

Comment:
@eliasjpr - What do you think about the unit test in the code below?

With the other controller tests, those are testing the class directly. I think they are more helpful, but strictly speaking, they may fall under "feature" tests or some other category.

I've included both here - since we haven't really fleshed out how to test controllers and organize those tests.